book/graphs/implementation.md

```diff
@@ -31,7 +31,10 @@ class Vertex(object):
         self.neighbors[neighbor] = weight
 
     def __str__(self):
-        return '{} neighbors: {}'.format(self.key, [x.key for x in self.neighbors])
+        return '{} neighbors: {}'.format(
```

Comment:
Did this change actually affect spillover in those boxes? Or does this address character limits in a text editor?